docs/extending/models.md

```diff
@@ -40,6 +40,22 @@ Kiota [has plans](https://github.com/microsoft/kiota/issues/648) to support disc
 
 In addition to all the described properties, a model will contain a set of default members.
 
+### Factory method
+
+The Factory method (static) is used by the Parse Node implementation to get the base or derived instance according to the discriminator value. If an operation describes returning a Person model, and the Person model has discriminator information (mapping + property name), and the response payload contains one of the mapped value (e.g. Employee), the deserialization process will deserialize the the derived Employee type instead of the base Person type. This way SDK users can take advantage of the properties that are defined on this specialized model.
+
+```CSharp
```

Comment:
use `csharp` instead of `CSharp` for compatibility with Docs.ms (future-thinking)

Reply:
```suggestion
```csharp
```

Thanks... habbits...